src/rips_scan_tag.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+TAG_NAME="$(curl -s https://api.github.com/repos/greenpeace/${RIPS_SCAN_REPO}/tags | jq -r '.[0].name')"
+TAG_COMMIT="$(curl -s https://api.github.com/repos/greenpeace/${RIPS_SCAN_REPO}/tags | jq -r '.[0].commit.sha')"
+
+APPLICATION_ID=${RIPS_APPLICATION_ID}
+
+echo "Downloading zip archive for tag: ${TAG_NAME}"
+
+wget https://github.com/greenpeace/planet4-master-theme/archive/${TAG_COMMIT}.zip
+
+wget https://github.com/rips/rips-cli/releases/download/1.2.1/rips-cli.phar -O ./rips-cli
```

Comment:
Is it worth adding this to the builder image instead of downloading each time? IDK

Reply:
+1, I guess it reduces the "error surface" (?)

Reply:
Actually, on second thoughts, as this repo doesn't have other compiled dependencies on it, feels a bit weird to add it. I would just leave as is for now.